cert.go

```diff
@@ -331,6 +331,14 @@ func (c *Certificate) AddExtension(nid NID, value string) error {
 	return nil
 }
 
+// AddCustomExtension add custom extenstion to the certificate.
+func (c *Certificate) AddCustomExtension(nid NID, value []byte) error {
+	if int(C.add_custom_ext(c.x, C.int(nid), (*C.char)(C.CBytes(value)), C.int(len(value)))) == 0 {
```

Comment:
Aren't we copying `value` inside the C code? If so, I think we need to either free the copied string after calling this, or call `(*C.char)(unsafe.Pointer(&value[0]))` instead of `C.CBytes`.

Note: I'm not an expert in CGO so I really have no idea what's safe and what's not safe.

Reply:
I've freed the memory